pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md.erb

```diff
@@ -42,6 +42,15 @@ Computer Science Discoveries is an introductory course for 6 - 10th grade studen
 <br>
 <hr>
 
+# Creating Apps for Devices
+
+Every day, we use devices — computers, phones, and more — that help us accomplish tasks both big and small. When students learn to program these devices, foundational computer science comes alive! In this module, which spans roughly 6 weeks, students will learn the fundamentals of programming devices.
+
+[<button>Learn about Maker</button>](/maker)&nbsp;&nbsp;[<button>Explore the Creating Apps for Devices Module</button>](https://studio.code.org/s/devices)&nbsp;&nbsp;
```

Comment:
I'm worried about the use of `&nbsp;&nbsp;`, but I think you know more about this than I do. Does this adjust correctly for mobile too? I'm assuming you're doing this to ensure these buttons are on the same line?

Reply:
I copied the same pattern used for the other similar sections on this page that are using Markdown, but I can confirm they look good on mobile. (I agree though, it's not ideal!)